app.py

```diff
@@ -14,6 +13,30 @@
 async def start(request):
     return text("pong")
 
+@app.route("/matrix", methods=["GET", "POST"])
+@requires_db
+async def matrix(db, request):
```

Comment:
Lets call this one `list_ranking_matrices(db, request)` and limit the endpoint to `GET` requests.